Scripts/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends":"eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 11
```

Comment:
Maybe it's better to clarify why you choose ecmaVersion 11?

Reply:
that's the smalest number that didn't raise any error when coding. This number might change when the minification will be added but the odds are small.